### PR TITLE
Some bugfixes to incorporate

### PR DESCRIPTION
**Pull request by GaretJax**
I'm using your branch on CMS v4 and I have encountered some issues that I have ~~fixed~~ worked around with the following changes.

It is meant to serve as inspiration/bug report, so feel free to rework and make yours.

### CODE REVIEW

File: djangocms_blog/models.py [hunk 1]
```diff
@@ -578,6 +578,13 @@ class Meta:
     objects = GenericDateTaggedManager()
     admin_manager = AdminDateTaggedManager()
 
+    @property
```
Comment (fsbraun) [inline djangocms_blog/models.py]: I do not think this should be needed? When did you encounter a connected issue?

**Comment by fsbraun** [comment]
Hey @GaretJax ! Thanks a lot!! With the exception, of the `in_navigation` property, I would like to merge this. When/why do you think this is needed?

**Comment by GaretJax** [comment]
Mainly because it was raising an exception without :-D

I will have to try and see if it still happens if I remove that code again, and report back with more details. This will take some time however, I'm far from my computer for a couple of weeks.

**Comment by fsbraun** [comment]
If it still raises an exception without `in_navigation`, can you share the traceback?

File: djangocms_blog/models.py (hunk 1 above)
Comment (fsbraun) [reply djangocms_blog/models.py]: This turns out to be a bug in djangocms-versioning. Fixed here: https://github.com/django-cms/djangocms-versioning/pull/423

**Review by fsbraun — APPROVED** [review]

**Pull request merged**